modules/table/src/main/java/org/apache/ignite/internal/table/distributed/index/IndexUtils.java

```diff
@@ -84,32 +87,34 @@ public static void registerIndexToTable(
      * @param catalogService Catalog service.
      * @param partitionSet Partitions for which index storages will need to be created if they are missing.
      * @param schemaRegistry Table schema register.
+     * @param lwm Low watermark.
      */
     public static void registerIndexesToTable(
             TableViewInternal table,
             CatalogService catalogService,
             PartitionSet partitionSet,
-            SchemaRegistry schemaRegistry
+            SchemaRegistry schemaRegistry,
+            @Nullable HybridTimestamp lwm
```

Comment:
Please describe in the javadoc why this argument is used and how it will be used.